tempesta_fw/t/unit/test_tfw_str.c

```diff
@@ -1178,6 +1178,292 @@ TEST(tfw_str_collect_cmp, collect_chunks)
 	EXPECT_EQ(TFW_STR_CHUNKN(&out), 5);
 }
 
+TEST(tfw_str_add_compound, plain)
+{
+	TfwStr *s;
+	TfwStr s1 = {
+		.len = 16,
+		.ptr = "abcdefghijklmnop"
+	};
+	TfwStr s2 = {
+		.len = 10,
+		.ptr = "qrstuvwxyz"
+	};
+
+	s = tfw_str_add_compound(str_pool, &s1);
```

Comment:
The `s1` string is allocated on stack, but `tfw_str_add_compound()` can use `realloc` operations. The code is dangerous, and can have side effects. Please, use `TFW_STR()` or `make_plain_str()` functions from `tfw_str_helper.h`. Same applies to other test cases.